### PR TITLE
add trending topic lex, wire in agent

### PR DESCRIPTION
**Pull request by haileyok**

### CODE REVIEW

File: packages/bsky/src/api/app/bsky/graph/getTrendingTopics.ts [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+  SkeletonFnInput,
+} from '../../../../pipeline'
+import { HydrateCtx, Hydrator } from '../../../../hydration/hydrator'
+import { Views } from '../../../../views'
+import { QueryParams } from '../../../../lexicon/types/app/bsky/unspecced/getTrendingTopics'
+import AtpAgent from '@atproto/api'
+import { TrendingTopic } from '../../../../lexicon/types/app/bsky/unspecced/defs'
+import { InternalServerError } from '@atproto/xrpc-server'
+
+export default function (server: Server, ctx: AppContext) {
```
Comment (dholms) [inline packages/bsky/src/api/app/bsky/graph/getTrendingTopics.ts:18]: gotta mount this route in `src/api/index.ts`

File: packages/bsky/src/context.ts [hunk 2]
```diff
@@ -52,6 +53,10 @@ export class AppContext {
     return this.opts.suggestionsAgent
   }
 
+  get topicsAgent(): AtpAgent | undefined {
```
Comment (dholms) [inline packages/bsky/src/context.ts:56]: is this going out to Discover? tempted to just re-use the suggestionsAgent in the meantime, unless we have a reason to introduce a new one

not very concerned either way though. Happy to keep it as is as well

**Review by dholms — APPROVED** [review]
looks great 🙌

File: packages/bsky/src/api/app/bsky/graph/getTrendingTopics.ts [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+  const getTrendingTopics = createPipeline(
+    skeleton,
+    hydration,
+    noBlocksOrMutes,
+    presentation,
+  )
+  server.app.bsky.unspecced.getTrendingTopics({
+    auth: ctx.authVerifier.standard,
```
Comment (dholms) [inline packages/bsky/src/api/app/bsky/graph/getTrendingTopics.ts:26]: oh, do we want this to be able to be hit unauthed? if so, we can use `standardOptional` here

File: packages/bsky/src/context.ts (hunk 2 above)
Comment (haileyok) [reply packages/bsky/src/context.ts:56]: have it on a separate box rn which is why i created the separate agent. we might combine it sooner than later at which point we could def drop this.

File: packages/bsky/src/api/app/bsky/graph/getTrendingTopics.ts (hunk 3 above)
Comment (haileyok) [reply packages/bsky/src/api/app/bsky/graph/getTrendingTopics.ts:26]: ah nice, ty

**Pull request merged**